src/test/java/com/iexec/worker/pubsub/SubscriptionServiceTests.java

```diff
@@ -79,24 +73,27 @@ void shouldNotSubscribeToExistingTopic() {
         assertThat(subscriptionService.isSubscribedToTopic(CHAIN_TASK_ID)).isTrue();
         verify(stompClientService, atMost(1)).subscribeToTopic(anyString(), any());
     }
+    // endregion
 
+    // region unsubscribe
     @Test
     void shouldUnsubscribeFromTopic() {
         when(stompClientService.subscribeToTopic(anyString(), any())).thenReturn(Optional.of(subscription));
 
         subscriptionService.subscribeToTopic(CHAIN_TASK_ID);
         assertThat(subscriptionService.isSubscribedToTopic(CHAIN_TASK_ID)).isTrue();
-        subscriptionService.unsubscribeFromTopic(CHAIN_TASK_ID);
-        assertThat(subscriptionService.isSubscribedToTopic(CHAIN_TASK_ID)).isFalse();
+        assertThat(subscriptionService.purgeTask(CHAIN_TASK_ID)).isTrue();
         verify(subscription, times(1)).unsubscribe();
```

Comment:
`times(1)` is implied when not mentioned
```suggestion
        verify(subscription).unsubscribe();
```

Reply:
Resolve in [36484b8](https://github.com/iExecBlockchainComputing/iexec-worker/pull/620/commits/36484b81174020d317b49d1b9ac290f957a192f9)